Setup Internal Feeds (centOSStreamContainer) fails with "PowerShell exited with code '1'"

**Repro Pipeline:**
#[20260126.22](https://dev.azure.com/dnceng/internal/internal%20Team/_build/results?buildId=2888230&view=logs&j=0e2e93c6-d6eb-5863-89e0-74051c706661&t=857872b8-e8b4-500f-ad11-5b9de896dc7b&l=219)

#[20260126.15](https://dev.azure.com/dnceng/internal/internal%20Team/_build/results?buildId=2888030&view=logs&j=0e2e93c6-d6eb-5863-89e0-74051c706661&t=857872b8-e8b4-500f-ad11-5b9de896dc7b&l=219)

**Repro Branch:**
internal/10.0.2xx
10.0.2xx
10.0.3xx

**Error Message:**
##[error]PowerShell exited with code '1'.
Finishing: Setup Internal Feeds (centOSStreamContainer)

--- a/eng/pipelines/templates/jobs/vmr-build.yml
+++ b/eng/pipelines/templates/jobs/vmr-build.yml
@@ -452,19 +452,6 @@ jobs:
         rm -f $(sharedComponentsDir)/Private.SourceBuilt.SharedComponents.*.tar.gz
       displayName: Download Shared Component Artifacts
 
-  # https://github.com/dotnet/dotnet/issues/1758
-  # After a re-boostrap:
-  # - Uncomment the line below, replacing the one after it.
-  # - Uncomment the two parameter lines at the end of the template parameters.
-  # - ${{ if and(eq(parameters.sign, 'True'), ne(parameters.signType, 'DryRun')) }}:
-  - ${{ if and(eq(parameters.sign, 'True'), eq(parameters.signType, 'Real')) }}:
-    - template: ${{ variables['Build.SourcesDirectory'] }}/eng/common/core-templates/steps/install-microbuild.yml
-      parameters:
-        enableMicrobuild: true
-        enableMicrobuildForMacAndLinux: true
-        # ${{ if ne(parameters.signType, 'Real') }}:
-        #   microbuildUseESRP: false
-
   - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
     - task: PowerShell@2
       displayName: Setup Internal Feeds
@@ -496,6 +483,20 @@ jobs:
     - task: NuGetAuthenticate@1
       inputs:
         nuGetServiceConnections: devdiv/engineering, devdiv/dotnet-core-internal-tooling
+
+  # https://github.com/dotnet/dotnet/issues/1758
+  # After a re-boostrap:
+  # - Uncomment the line below, replacing the one after it.
+  # - Uncomment the two parameter lines at the end of the template parameters.
+  # - ${{ if and(eq(parameters.sign, 'True'), ne(parameters.signType, 'DryRun')) }}:
+  - ${{ if and(eq(parameters.sign, 'True'), eq(parameters.signType, 'Real')) }}:
+    - template: ${{ variables['Build.SourcesDirectory'] }}/eng/common/core-templates/steps/install-microbuild.yml
+      parameters:
+        enableMicrobuild: true
+        enableMicrobuildForMacAndLinux: true
+        # ${{ if ne(parameters.signType, 'Real') }}:
+        #   microbuildUseESRP: false
+
 
   - ${{ if eq(parameters.targetOS, 'windows') }}:
     # Node 20.x is a toolset dependency to build aspnetcore

